MAUI/ImageEditor/getting-started.md

```diff
@@ -307,4 +307,97 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
 {% endhighlight %}
 {% endtabs %}
 
-N> The size value will only be available after the image has been loaded into view.
\ No newline at end of file
+N> The size value will only be available after the image has been loaded into view.
+
+### Check Image edited

```

Comment:
### Check the Edited Image

Reply:
@florenceliliana the context here is **whether the image has been edited or not.**

Please suggest headline based on that.